pages/header.page.ts

```diff
@@ -0,0 +1,11 @@
+import { type Locator, Page } from '@playwright/test';
+
+export class HeaderPage{
+    readonly page: Page;
+    readonly button_logout: Locator;
+   
+    constructor(page){
+        this.page = page;  
+        this.button_logout = page.getByTestId('nav-logout-button');
+    }
```

Comment:
Hat mit dem PW Reset nichts zu tun, aber das Logo oben links soll auch klickbar sein und zur Startseite führen. Kommt das in einem anderen Scope?

Reply:
Das kommt ebenfalls mit einem anderen Testfall.